solutions/kubernetes/cortx-perfline.sh

```diff
@@ -0,0 +1,194 @@
+#!/bin/bash
```

Comment:
Please move scripts to https://github.com/Seagate/cortx-re/tree/main/scripts/performance